src/ClientData/SharedTestConstants.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#ifndef CLIENT_DATA_SHARED_TEST_CONSTANTS_H_
+#define CLIENT_DATA_SHARED_TEST_CONSTANTS_H_
+
+#include <array>
+#include <cstddef>
+#include <cstdint>
+
+#include "ClientData/TimerTrackDataIdManager.h"
+
+namespace orbit_client_data {
+
+constexpr size_t kTimersForFirstId = 3;
```

Comment:
Hmm, I am not convinced deduplication is a real goal for Tests here. For me, I took quite a bit of scrolling back and forth when reviewing the CaptureDataTest to notice, that not all the constants are defined in that file. Not sure if this is really a big deal, but I would probably find it more readable if a tests is mostly self-contained.

Reply:
OK, I've inlined the variables.